fast64_internal/f3d/f3d_writer.py

```diff
@@ -1613,6 +1700,25 @@ def saveOrGetF3DMaterial(material, fModel, obj, drawLayer, convertTextureData):
 
     return fMaterial, texDimensions
 
+def getCelShadingInfo(material):
```

Comment:
to me it seems like `CelShadingInfo` is used enough where it could warrant some type of class or definition, as well as the levels. then we can add type annotations everywhere and keep things in sync easier

Reply:
Agreed, I'll add that.